pages/admin/create-class.tsx

```diff
@@ -0,0 +1,271 @@
+        "classImage",
+        "https://www.planetware.com/wpimages/2020/02/france-in-pictures-beautiful-places-to-photograph-eiffel-tower.jpg",
+    );
+
+    const [className, setClassName] = useLocalStorage("className", "");
+    const [associatedProgram, setAssociatedProgram] = useLocalStorage("associatedProgram", "");
+
+    const [teacherName, setTeacherName] = useLocalStorage("teacherName", "");
+    const [dateAvailable, setDateAvailable] = useLocalStorage("dateAvailable", "");
+
+    const [location, setLocation] = useLocalStorage("location", "");
+    const [locationDescription, setLocationDescription] = useLocalStorage(
+        "locationDescription",
+        "",
+    );
+
+    const [recurrence, setRecurrence] = useLocalStorage("recurrence", "");
+    const [repeatOn, setRepeatOn] = useLocalStorage("repeatOn", "");
+    const [endRecurrence, setEndRecurrence] = useLocalStorage("endReuccrence", "");
+
+    const [age, setAge] = useLocalStorage("age", "");
+    const [aboveUnder, setAboveUnder] = useLocalStorage("aboveUnder", "");
+    const [maxCapacity, setMaxCapacity] = useLocalStorage("maxCapacity", "");
+    const [price, setPrice] = useLocalStorage("price", "");
+
+    const [classDescription, setClassDescription] = useLocalStorage("classDescription", "");
+
+    const [stripeLink, setStripeLink] = useState("");
+
+    async function save() {
+        //Create Stripe product and price
+        const product = await stripe.products.create({
+            name: className,
+        });
+
+        const stripePrice = await stripe.prices.create({
+            unit_amount: parseInt(price),
+            currency: "usd",
+            recurring: { interval: "month" }, //recurrance?
+            product: product.id,
+        });
+
+        const data = {
+            name: className,
+            programId: associatedProgram,
+
+            startDate: null,
+            endRecurrence: endRecurrence,
+
+            weekday: repeatOn,
+
+            borderAge: age,
+            isAgeMinimal: aboveUnder === "under", //Assuming that minimal means "at least"
+
+            stripePriceId: stripePrice.id,
+        };
+        //Save to database
+        const request = {
+            method: "POST",
+            headers: { "Content-Type": "application/json" },
+            body: JSON.stringify(data),
+        };
+        const response = await fetch("/api/program", request);
+        //TODO: handle error?
+    }
+
+    return (
+        <Wrapper>
+            <Heading
+                alignSelf="flex-start"
+                fontWeight={700}
+                fontSize={22}
+                style={{ color: colourTheme.colors.Blue, margin: 25 }}
+            >
+                Create
+            </Heading>
+            <Box
+                style={{
+                    borderBottom: "2px solid lightgray",
+                    width: "100%",
+                    marginBottom: 50,
+                }}
+            ></Box>
```

Comment:
We actually already have a component for this!
```suggestion
    <AdminHeader>Create</AdminHeader>
```